docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -49,11 +51,24 @@ \section{Team Member Roles}
 
 \section{Workflow Plan}
 
-\begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classificaiton, etc.?
-\end{itemize}
+We will be using git as a version control software with GitHub as our remote repository. The root branch will be \code{master} and development branch mased on \code{master} will be \code{dev}. The developer will branch off from \code{dev} to work on issues. There will be a new branch for each issue and will be named comtaining the issue number and name detailed below. Continuous Inntegration is set up between all branches \code{dev}, \code{master}, and Issues.
+
+Issues are used to track tasks and meetings related to the project. Meeting issues will be created before each meeting according to the \textbf{MeetingIssueTemplate.md} template. These include addentance, minutes, and action items from the meeting. These notes can be referenced from other issues and PRs for tracability. 
+Feature issues are used to track tasks. They can be referenced in PRs and meetings and outline the tesk that the developer must fullfill in their PR. 
+Issues can be assigned to a Milestone which tracks issues related to deliverables. These are useful for filtering issues and also tracking due dates.
+Issues can be assigned labels which give information of which part of the project the issue related to. These can denote issue types as well as issue severity or complexity. Useful for filtering.
+
+\subsection{Version Control Developer Step-By-Step}
+
+\begin{enumerate}
+	\item Granted Issue ticket with issue number.
+	\item Open branch with name \{IssueNumber\}/\{issue-name\}. (eg. \code{135/add-user-endpoint})
+	\item Commit with format: \{IssueNumber\}:\{commit message\}. (eg. \code{"135: updated yaml for add-user-endpoint"})
+	\item Open Pull Request on GitHub following the \textbf{PRTemplate.md} template, thus mapping the PR to the Issue.
+	\item Await 2 reviewers.
+	\item Fix reviewer comments.
+	\item Merge to dev.
+\end{enumerate}
```

Comment:
Maybe add a comment where after some number of updates or milestone is achieved, master will be updated at the end?

Reply:
Additionally, here's the rubric entry for this section, just as a reference for everyone reviewing:

![image](https://github.com/r-yeh/grocery-spending-tracker/assets/67233377/6f34d466-32d9-42f4-b694-9b75c55680a7)